src/views/UserInput.js

```diff
@@ -0,0 +1,28 @@
+import { Console } from "@woowacourse/mission-utils";
+import RACING_VARIABLES from "../constants/RacingVariables.js";
+
+class UserInput {
+
+  async getUserInputCars() {
+
+    try {
+      const userInput = await Console.readLineAsync(RACING_VARIABLES.INPUT_PROMPT);
+      return userInput;
```

Comment:
```suggestion
      return Console.readLineAsync(RACING_VARIABLES.INPUT_PROMPT);
```
여기는 별다른 로직이 없어서 바로 `return`해도 무방할 것 같네요~!